source/Halibut.Tests/ProxyFixture.cs

```diff
@@ -44,6 +49,9 @@ public async Task OctopusCanNotSendMessagesToTentacle_WithEchoService_AndABroken
             {
                 await clientAndService.HttpProxy!.StopAsync(CancellationToken.None);
 
+                // var echo = clientAndService.CreateClient<IEchoService, IAsyncClientEchoService>();
```

Comment:
??

Reply:
This is the async version of the test, which doesn't work. Will remove